docs/en/sql-reference/10-sql-commands/00-ddl/17-dictionary/create-dictionary.md

```diff
@@ -0,0 +1,91 @@
+---
+title: CREATE DICTIONARY
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.636"/>
+
+Creates a dictionary using a specified source.
+
+## Syntax
+
+```sql
+CREATE DICTIONARY <dictionary_name>
+(
+    <column_name1> <data_type1>,
+    <column_name2> <data_type2>,
+    ...
+)
+PRIMARY KEY <primary_key_column>
+SOURCE(<source_type>(<source_parameters>))
+```
+
+| Parameter              | Description                                                                                                                                |
+|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `<dictionary_name>`    | The name of the dictionary.                                                                                                                |
+| `<column_name>`        | The name of a column in the dictionary.                                                                                                    |
+| `<data_type>`          | The type of data stored in the column.                                                                                                     |
```

Comment:
data type can specify a default value, if a source value not exist, the default value will be returned instead.